pkgs/development/python-modules/keras-tuner/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, colorama
+, scikitlearn
+, future
+, tqdm
+, scipy
+, requests
+, tabulate
+, terminaltables
+, tensorflow_2
+, tensorflow-tensorboard_2
+ }:
+
+buildPythonPackage rec {
+  pname = "keras-tuner";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "a9626842bc032bb0c8f3152bbc90910f50db3221f8aa980ec82ac729692707ec";
+  };
+
+  propagatedBuildInputs = [
+    colorama
+    scikitlearn
+    future
+    tqdm
+    scipy
+    requests
+    tabulate
+    terminaltables
+    tensorflow_2
+    tensorflow-tensorboard_2
+  ];
+  checkInputs = [
+    pytest

```

Comment:
Could you please check if it's possible to use `pytestCheckHook`?

Reply:
Done